dev/com.ibm.ws.webcontainer.servlet.4.0_fat/fat/src/com/ibm/ws/fat/wc/FATSuite.java

```diff
@@ -73,7 +74,7 @@
  */
 @RunWith(Suite.class)
 @SuiteClasses({
-                // WCPushBuilderSecurityTest.class,
+                WCPushBuilderSecurityTest.class,
```

Comment:
I'm assuming uncommenting this was accidental?

Reply:
Verify in integration -- may be commented out already?

Reply:
comment out